DigitalOffice/Tests/UserServiceTests/Repositories/UserRepositoryTests.cs

```diff
@@ -80,6 +80,22 @@ public void ShouldReturnUserIfUserWithRequiredIdExists()
             SerializerAssert.AreEqual(dbUser, resultUser);
         }
 
+        [Test]
```

Comment:
I think the tests in the repository need to be separated somehow, for example, using regions.

Reply:
Agreed. Done